gnosis/safe-react
# Bugfix - Trimming leading/trailing spaces when saving or editing AB entries

## What it solves
This PR closes #2417 

## How this PR fixes it
Any leading/trailing spaces are trimmed before saving it into the AB. 

## How to test it
1. Open/Load a safe
2. Go to Address book
3. Create or edit an AB entry adding some leading/trailing spaces to the owner name.
4. Name should be saved without those spaces.
